### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by rainy0niu** [trevormcdonald/VVIT#9]
The project is aiming to validate the value investing theory, and examine whether investing according to the value investing strategy will bring a positive return. This is certainly an ambitious goal. I was expecting to read a profound report after reading the introduction. However, I am surprised by the project and report quality in general. 

To start with, the first half of the first paragraph of data exploration section is directly **plagiarizing** the content from the website [investopedia](http://www.investopedia.com/terms/u/undervalued.asp#ixzz4S1sqNOsI):

"An undervalued stock is believed to be priced too low based on current indicators, such as those used in a valuation model. Should a particular company’s stock be valued well below the industry average, it may be considered undervalued. In these instances, value investors may focus on acquiring these investments as a method of pulling in reasonable returns for a lower initial cost."

I understand that this is mostly a definition, but the later half is also an explanation. I do think that copying directly word-by-word requires at least a reference to the source.  

In addition, the report surfers greatly from grammatical error. Some examples I spotted from the first two pages are:

1. our dataset _will_ contain all public companies with their current…
2. to _meansure_...
3. The first step… _is how_ to deal with…
4. Second, we used the strategy to exclude _the whole row which_ includes missing data.

The unprofessional formatting of including unexplained screenshots of output from a language that are badly cropped is also obvious in the report. As a result, the report looks more like a badly written homework assignment than a scholarly paper. 

--------

In terms of contents of this report, there are several things that team might consider exploring if they are continuing with this research.

1. Have the team consider merging data from a different source? I noticed that the team mentioned dropping data due to missingness. More specifically, the number of stocks analyzed dropped from 3700 to 1421 stocks, and then to 200. Similarly, the team decided to drop some of the features in the dataset due to the same reason. I understand there are technical difficulties related to obtaining some of the data. However, there are multiple daily stock data open to download via web. For example, Yahoo Finance preserves decent data dating at least back to 1910. I also know that R has packages that allows you to download maintained stock data in a good format. 

2. Could you explain SVM a bit more in the report? I don't think this is a technique covered in class, and the audiences might need some background information.

3. Price change is a good idea - maybe not daily? As the report mentioned using SVM to classify the sign of daily stock return change, I really appreciate the idea. It make sense that as you are trying to validate VVIF, you should show whether investing in the undervalued stocks gives investor a "big" return. However, I assume it is a rather long term investment for the investors following VVIF. Maybe using the return difference for a longer time frame is better?

4. It seems like you are using a linear model of return in terms of different statistics, but no offset. Is there a reason you did not include an offset in the second model, but in the first section?

5. It looks like your features are in different magnitudes. Have you considered scaling them and then fit?

6. Is the error you mentioned on test set or training set?

7. The question that the team claimed to solve is rather broad, but it is not clear how the models they described are proving/disproving VVIF. For one thing, what is a "big" return? How would the return as a function of the real time statistics be an analysis on how well they predict future returns?
